### DISCUSSION TITLE
How do I use Algolia search in the docs? I'm confused!

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1126] [Q&A]
I've gone through the docs, but I still don't understand how to set up Algolia search.

Do I need to create files like `scripts/update-index.mjs` and `api/search/route.js`? What other steps are needed after that?

Also, how many files do I actually need to make? Is there a clear explanation or maybe an example, like in FumaDocs, where Algolia is already set up?

**Comment by fuma-nama** [comment]
Updated the docs, is there still anything unclear to you?
